src/utils/messages.js

```diff
@@ -73,6 +73,12 @@ export const UPDATE_REMINDER_MESSAGE = chalk.yellow(
     "Run `quest update` to download update.\n"
 );
 
+export const BREAKING_CHANGE_MESSAGE = chalk.yellow(
+  "Deprecated version of quest detected in local repo. New version introduces breaking changes.\n" +
+  "Run `quest update` to download breaking update.\n") +
```

Comment:
`quest update` does not modify any quests.
To download breaking updates for the quest, they will have to run:
1. `quest find {QUEST_NAME}`